Edit Dashboard Description

aboydnw opened the pull request:
Recreating this pull request

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
 
-The [VEDA Dashboard](https://www.earthdata.nasa.gov/dashboard/) is one of several user interfaces in the VEDA project. The VEDA Dashboard is a publishing platform for results of scientific analysis of Earth data at NASA, 
-in the form of Dataset information pages ("Datasets") and stories ("Data Stories").
+The VEDA Dashboard is an open-source set of configurable tools and pages that enable you and your team to visualize, analyze, and communicate science data. Learn more about the VEDA Dashboard set of tools below. If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit this page.
```
```suggestion
The VEDA Dashboard is an open-source set of configurable tools and pages that enable you and your team to visualize, analyze, and communicate science data. 

Learn more about the principles the VEDA Dashboard is designed around below.

If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit the [VEDA UI docs](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/development/SETUP.md).
```

Was this the page you meant to link to?

How would you describe the content of this page? I changed the sentence above to "the principles the VEDA Dashboard is designed around".

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
+The VEDA Dashboard is an open-source set of configurable tools and pages that enable you and your team to visualize, analyze, and communicate science data. Learn more about the VEDA Dashboard set of tools below. If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit this page.
 
-It also is aligned with VEDA's goal of advancing interactive visualization and exploratory visual data analysis. This dashboard constitutes:
+### 1. Science teams need to present their research program, findings, and methods and data to their stakeholders, colleagues, and other audiences
```
```suggestion
### Who needs VEDA Dashboards?

_Science teams need to present their research program, findings, and methods and data to their stakeholders, colleagues, and other audiences_
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
-* an evolution of COVID-19 dashboard to provide the interactive storytelling for various environmental changes using Earth observation data;
-* a transformation of high-valued NASA datasets to dynamic visualization, enabling users to perform on-the-fly visual analysis; support for environmental justice initiatives through the integration of NASA Earth observation data and socio-economic data;
-* a tool by which users can visually explore and localize data, and perform independent analysis; data-driven stories that are exportable in multiple formats; and a situational awareness system that brings together Earth observation datasets such as greenhouse gasses, air quality, and sea level rise, among others.
+### 2. Visualizing spatio-temporal science data in the browser is still a challenge that off-the-shelf content management systems have not solved
```
```suggestion
### How is VEDA Dashboard different from Content Management Systems?

_Visualizing spatio-temporal science data in the browser is still a challenge that off-the-shelf content management systems have not solved_
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
+- The VEDA Dashboard set of tools allows you to visualize spatio-temporal data assets within your website, maintaining an interoperable reference to the source data. At the same time, it is completely open source, customizable, and comes without a subscription.
 
-![Dashboard Screenshot](images/dashboard.png)
+### 3. We show that, with cloud-optimized data, a platform can enable science workflows AND communicates to stakeholders
```
```suggestion
### Advantages of cloud-native geospatial technologies

_We show that, with cloud-optimized data, a platform can enable science workflows AND communicate to stakeholders_
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
+Another benefit of dynamic tiling is that users are able to adjust viewing parameters such as color map and value range to focus on, to study details of their interest.
 
-## Contributing
+### 4. Dashboard features are built and optimized for Earth science data but might be useful in other similar applications
```
```suggestion
### Many areas of data-intensive research can benefit from VEDA Dashboard

_Dashboard features are built and optimized for Earth science data but might be useful in other similar applications_
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
 
-Please see our docs on [Dashboard Configuration](../contributing/dashboard-configuration/index.qmd).
\ No newline at end of file
+### 5. The VEDA Dashboard is completely open-source
```
```suggestion
### The VEDA Dashboard is completely open-source
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
+- The code is open and accessible, and the tool itself proliferates the use of open standards and reproducible data access for open science
+Community involvement will become important to the long-term success of VEDA
+
+### 6. The number of websites that use VEDA UI is growing, each with different content and data, site layouts, and selection of features
```
```suggestion
### What drives the evolution of this software project?

_The number of websites that use VEDA UI is growing, each with different content and data, site layouts, and selection of features_
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
+Planned VEDA instances:
+- Air Quality Portal
+
+### 7. There are a wide variety of potential users of the VEDA Dashboard, although each science team usually narrows down on one or two groups for their instance
```
```suggestion
### Who are the target users of VEDA Dashboard?

_There are a wide variety of potential users of the VEDA Dashboard, although each science team usually narrows down on one or two groups for their instance_
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
+### 7. There are a wide variety of potential users of the VEDA Dashboard, although each science team usually narrows down on one or two groups for their instance
+<Brian’s graphic of different target users>
+
+### 8. VEDA UI provides a suite of tools, widgets, and pages for science teams to create their own websites, rather than a replicable white-label website/dashboard
```
```suggestion
### How can I use VEDA UI to power a web presence?

_VEDA UI provides a suite of tools, widgets, and pages for science teams to create their own websites, rather than a replicable white-label website/dashboard_
```

j08lue commented on nasa-veda-platform/dashboard/index.qmd:
```diff
@@ -1,21 +1,64 @@
+A template instance is provided to show how these tools and widgets could be configured to form a cohesive site that showcases science data.
+<Link to VEDA UI component library docs>
+
+### 9. Through VEDA's Data Services, utilizing an integrated STAC, a VEDA Dashboard can show data from a variety of sources
```
```suggestion
### What data can be integrated into VEDA Dashboard?

_Through VEDA's Data Services, utilizing an integrated STAC, a VEDA Dashboard can show data from a variety of sources_
```

j08lue reviewed (changes requested):
I think these headings are very long. Can we rewrite them as Q&A like questions?

We need to insert some of the diagrams and references that the text mentions.

aboydnw commented:
Thanks @j08lue, I addressed your comments to make the titles shorter. I'd like to keep the structure where it's a bit more like a list than an FAQ page. We can always change it later if it's not working.

I also removed placeholders for links or images, we can add those later too

j08lue reviewed (approved):
Great addition to the docs!
